### ISSUE TITLE
Handle Squads More Than 4 Players in Teams

### ISSUE THREAD

**Issue by binlong** [pubgsh/api#3]
There is an issue with matches not getting written when Custom Squads have more than 4 players on a team. In the Match.js on line 15 the max squad size is hard coded to 4. I believe this is part of the issue.

**Comment by binlong** [comment]
@apazzolini 
It seems the only purpose for team size is just to put the matches in the correct column for UI display. I would like to suggest to instead use gameMode and isCustomMatch to determine which column to put the matches under. Also add another column called Customs. This would then support all game modes with different team sizes

**Comment by binlong** [comment]
The issue with checking the team size for customs is you would have to actually look at LogMatchStart inside Telemetry. So, it is very heavy if you're just using it to organize which column to put the matches under. Also, what about team size of 3? Where would you put those matches under?

But if team size does not matter for Custom games then it's very easily to lump them all under Custom column since isCustomMatch attribute is accessible for Match object. This would be very easy to implement.

The reason I noticed this issue is because I'm on a Pro team and we have been scrimming with coaches as a 5th player in our games. Now we are not able to view our matches in pubg.sh due to the team size restriction. We used to heavily use your website to look at how the game is played out for every team during the scrim. It would be very helpful to all of us if we can utilize your great website again.

**Comment by apazzolini** [comment]
Indeed, I don't want to load Telemetry data just to see LogMatchStart, which is why I make a best-guess as to what the likely team size was.

The 5th man for scrimming is an interesting use case though, and I definitely want to support pro teams that use the site. I'll consider moving this up in priority.

**Comment by binlong** [comment]
I have created pull request as initial hack that would support Customs. Of course, I am not proficient in Javascript or graphql so there is most likely mistakes. 
API: https://github.com/pubgsh/api/pull/4
Client: https://github.com/pubgsh/client/pull/24

**Comment by apazzolini** [comment]
@binlong What player name and shard do you play on? I don't have any custom matches on my account right now and can't vet your changes.

**Comment by binlong** [comment]
@apazzolini 
Binlong/pc-na

**Comment by binlong** [comment]
Thanks a lot for the quick fix!
